pkg/cmd/factory/factory_impl.go

```diff
@@ -161,7 +161,7 @@ func (f factoryImpl) PdClient() (pd.Client, error) {
 
 	// TODO: we need to check all pd endpoint and make sure they belong to the same cluster.
```

Comment:
Maybe this comment became strange after this change?

Reply:
I still think it is necessary to leave this comment, because it means that we need to ensure that all PD are belong to a same cluster, but this function does not guarantee it now.

Reply:
I mean it might be weird to leave it here?

Reply:
done